examples/basic/src/VideoPlayer.tsx

```diff
@@ -201,15 +210,10 @@ const VideoPlayer: FC<Props> = ({}) => {
     });
   };
 
-  const channelUp = useCallback(() => {
-    console.log('channel up');
-    goToChannel((srcListId + 1) % srcList.length);
-  }, [srcListId]);
-
-  const channelDown = useCallback(() => {
-    console.log('channel down');
-    goToChannel((srcListId + srcList.length - 1) % srcList.length);
-  }, [srcListId]);
+  const onFullScreenExit = () => {
+    // iOS pauses video on exit from full screen
```

Comment:
When we exited fullscreen mode on iOS, the video was paused, and the state didn't update - to resume the video, we had to click the button twice instead of once

Reply:
Ok understood, but here you are hiding a bug in the package ?

Reply:
It's not a bug in package -> just in example app where we have "paused" or "playing" custom control and it wasn't updated after exiting fullscreen on ios